package.json

```diff
@@ -21,7 +21,6 @@
   },
   "lint-staged": {
     "*.{ts,tsx}": [
-      "tsc-files --noEmit",
```

Comment:
Add this back

Reply:
It doesn't work. I have talked with @vadim-kirpun for this

Reply:
If you are removing it then you also need to remove the dependency, although you should try and get it working instead.

Reply:
No it wont work. If using it in lint-staged, `tsconfig.json` will not be reachable. I am not removing the dep, as it is being used in precommit hook.

Reply:
[There are ways to get it working](https://github.com/okonet/lint-staged/issues/825), I have it on one of my personal private repos working...